app/web/src/components/Hero/Hero.tsx

```diff
@@ -13,153 +13,217 @@ import ProjectsCell from 'src/components/ProjectsCell'
+mountingHoleMoveIncrement=(hingeLength-2*mountingHoleEdgeOffset)/
+  (mountingHoleCount-1);
+
+module costomizerEnd() {}
+$fn=30;
+tiny=0.005;
+// modules
+module hingeBaseProfile() {
+  translate([pivotRadius,0,0]){
+    square([baseWidth,baseThickness]);
+  }
+}
+
+module hingeBodyHalf() {
+  difference() {
+    union() {
+      linear_extrude(hingeHalfExtrudeLength){
+        offset(1)offset(-2)offset(1){
+          translate([0,pivotRadius,0]){
+            circle(pivotRadius);
+          }
+          square([pivotRadius,pivotRadius]);
+          hingeBaseProfile();
+        }
+      }
+      linear_extrude(hingeLength){
+        offset(1)offset(-1)hingeBaseProfile();
       }
     }
-    return {
-      heroOffset: [-5, 0],
-      tutOffset: [4, 0],
-    }
-  }, [width])
+    plateHoles();
+  }
+}`.split('\n')
 
+export const Hero = () => {
   return (
     <div className="bg-ch-gray-800">
-      <ModelSection assetUrl="/coffee-lid.stl" offset={heroOffset} scale={0.06}>
-        <div className="grid lg:grid-cols-2 py-32">
-          <div className="flex items-end justify-center row-start-2 lg:row-start-1 pt-96 lg:pt-0 pr-12 pl-6">
-            <Link
-              to={routes.project({
-                userName: 'irevdev',
-                projectTitle: 'coffee-lid',
-              })}
-            >
-              <div className="grid grid-flow-col gap-2 sm:gap-4 items-center bg-ch-gray-760 bg-opacity-95 text-ch-gray-300 rounded-md p-2 font-fira-sans relative z-10 shadow-ch">
-                <div className="pl-1 sm:pl-4">
-                  <Gravatar
-                    image="CadHub/xvrnxvarkv8tdzo4n65u"
-                    className="w-12 h-12 mr-4"
-                    size={60}
-                  />
-                </div>
-                <div>
-                  <div className="text-xl sm:text-3xl">Coffee Lid</div>
-                  <div>IrevDev</div>
-                </div>
-                <div className="flex self-start">
-                  <CadPackage
-                    cadPackage="cadquery"
-                    className="px-3 py-1  sm:text-xl rounded transform translate-x-4 sm:translate-x-10"
-                  />
-                </div>
-              </div>
-            </Link>
+      <div className="relative h-0 w-0">
+        <svg viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <defs>
+            <clipPath id="code-blob-clip-path" clipPathUnits="objectBoundingBox" transform="scale(0.0038 0.0056)">
+              <path d="M68.5 169.159C13.3 167.159 5.69181e-05 144.659 0 71.1594C3.99994 13.1594 50.9244 -14.591 121.5 7.65941C223 39.6594 266 25.1594 263.5 113.659C261.634 179.701 191.5 173.616 68.5 169.159Z" fill="#C4C4C4"/>
+            </clipPath>
+          </defs>
+        </svg>
+      </div>
+      <div className="grid lg:grid-cols-5 max-w-8xl mx-auto">
+        <div className="relative row-start-2 col-start-1 h-full lg:row-start-1 lg:col-span-3 lg:col-start-1">
```

Comment:
lots of `grid` `start` `span` etc stuff since I'm forcing much of the content to overlap, and then needs to adjust for smaller screens too.